<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointConversorDialog</class>
 <widget class="QDialog" name="PointConversorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor de Ponto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Entrada:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="invertButton">
            <property name="text">
             <string>Inverter Entrada/Saída</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="convertButton">
            <property name="text">
             <string>Converter!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Saída:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="3">
              <widget class="QComboBox" name="outputSignedness">
               <item>
                <property name="text">
                 <string>Sem Sinal</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Complemento de 2</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Número:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="labelOutputSignedness">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Sinal:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="labelOutputPointPos">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Posição do Ponto:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="outputPointPos">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>63</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEditOutput">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="2">
              <widget class="QComboBox" name="outputFormat">
               <item>
                <property name="text">
                 <string>Notação Humana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Float 16 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Float 32 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Float 64 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ponto Fixo 8 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ponto Fixo 16 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ponto Fixo 32 Bits</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ponto Fixo 64 Bits</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0" rowspan="2">
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Formato:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Número:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Formato:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelInputSignedness">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sinal:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="lineEditInput"/>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="inputSignedness">
            <item>
             <property name="text">
              <string>Sem Sinal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Complemento de 2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="inputFormat">
            <item>
             <property name="text">
              <string>Notação Humana</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float 16 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float 32 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Float 64 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ponto Fixo 8 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ponto Fixo 16 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ponto Fixo 32 Bits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ponto Fixo 64 Bits</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="errorMessage">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
